rpc-server.py

```diff
@@ -62,6 +88,7 @@ def get_random_open_port():
 
 
 if __name__ == '__main__':
+    app.secret_key = bcrypt.gensalt().decode('utf8')
```

Comment:
The [secret_key](http://flask.pocoo.org/docs/0.11/api/#flask.Flask.secret_key) is required for using [sessions](http://flask.pocoo.org/docs/0.11/quickstart/#sessions), used to sign cookies (where the session ID is stored).

About [bcrypt](https://github.com/pyca/bcrypt/blob/8825f7fb639a8beadeb9d91e290eabd6a9e1f02c/src/bcrypt/__init__.py#L42), you're right, for this case it should be `os.urandom`. I'll change this.